contracts/l2/spell/initialize_bridge.cairo

```diff
@@ -12,32 +17,73 @@ namespace IBridge {
     }
 }
 
+@contract_interface
+namespace IProxy {
+    func set_implementation(implementation_hash: felt) {
+    }
+}
+
+@event
+func proxy_deployed(proxy_address: felt) {
+}
+
 @external
 func delegate_execute{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() {
+    // addresses to be set
+    const l2_governance_relay = 0;
+    const l2_bridge_address = 0;
+    const l1_bridge_address = 0;
+    const reward_token_address = 0;
+    const proxy_class_hash = 0;
+    const static_a_token_class_hash = 0;
+    const aUSDC = 0;
```

Comment:
nitpick: `a_dai` and `a_usdc`

Reply:
how about using camelCase for variables referring to l1 addresses?

Reply:
For Phase 2, I would prefer sticking with a single convention. Since this is Phase 1 code, and we haven't enforced such rules, it's fine to leave it like that.